src/draw/handler/Draw.Feature.js

```diff
@@ -13,13 +13,14 @@ L.Draw.Feature = L.Handler.extend({
 		if (options && options.shapeOptions) {
 			options.shapeOptions = L.Util.extend({}, this.options.shapeOptions, options.shapeOptions);
 		}
-		L.setOptions(this, options);
+
+		L.ToolbarAction.prototype.initialize.call(this, options);
```

Comment:
Hmmm. I'm not sure I agree about this tightly coupling the draw handlers to the toolbar. A toolbar action is not the same as a toolbar. Toolbar handlers are basically just ordinary Handlers, except that they've got toolbarIcon defaults defined in their options object, so that we don't have to check if options.toolbarIcon is undefined all the time. Instances of ToolbarAction also have a few toolbar-related methods - `_createIcon` and _addSubToolbar` - but these methods aren't called *until the toolbar action is added to a toolbar*. 

In particular, look at the code for `ToolbarAction#enable` and `#disable`. You'll see that the only difference between these methods, and those built in to `Leaflet.Handler` is that `ToolbarAction` checks to see if `#addHooks` and `#removeHooks` are actually defined before calling them, which I think is preferable behavior.

In short, even after the refactoring and the introduction of Leaflet.toolbar, Leaflet.draw handlers are perfectly happy to be called manually, outside of a toolbar, or embedded within a toolbar.